src/blocks/recommendations/index.php

```diff
@@ -0,0 +1,27 @@
+<?php
+/**
+ * The wp-parsely Recommendations Block entry file.
+ *
+ * @package Parsely
+ */
+
+/**
+ * Wrapper function to kick off the plugin. Sources & hooks initialization functions for the block & API classes.
+ *
+ * @return void
+ */
+function wp_parsely_recommendations_block_init() {
```

Comment:
What's the benefit to putting this in a function and then immediately calling the function? There are no locally scoped variables - in fact the only variable used is a global one.

Reply:
It's just a habit to put everything in a function. Functions are cheap :)

I'll see how it feels pulling it out.